test/BuyTshirt.spec.ts

```diff
@@ -1,44 +1,55 @@
-import { $, browser } from 'protractor';
+import { browser } from 'protractor';
+import { MenuContentPage, ProductListPage, ProductDetailPage, ProductAddedModalPage,
```

Comment:
In order to have more readability, you should only type a class for line
```
import {
  MenuContentPage,
  ProductListPage,
  ProductDetailPage,
  ProductAddedModalPage,
  ...,
  OrderResumePage } from '../src/page';
```

Reply:
done